package.json

```diff
@@ -52,7 +51,7 @@
     "gulp-tslint": "^6.1.1",
     "gulp-typescript": "^2.13.4",
     "gulp-typings": "^2.0.0",
-    "merge-stream": "^1.0.0",
+    "merge-stream": "^1.0.1",
```

Comment:
yea it does, there was an important fix released in `v1.0.1` to a bug that otherwise hides errors in our users build pipelines.